lib/truffle/ffi.rb

```diff
@@ -17,7 +17,7 @@ class DynamicLibrary
     RTLD_LOCAL  = Rubinius::Config['rbx.platform.dlopen.RTLD_LOCAL']
 
     def self.open(libname, flags)
-      code = libname ? "load #{libname}" : 'default'
+      code = libname ? "load '#{libname}'" : 'default'
```

Comment:
Do we need to escape quotes in `libname` as well then?

Reply:
I think that seems unlikely, but we might as well check it and raise an error since we have this restriction currently.

Reply:
I'm fairly certain this is only called when reading the `libs` file from our generated `.su` files. And we control the generation of those files. I suppose it's possible the Makefile has linker entries for paths with quotes though, and that's ultimately the source data used for the `libs` file.